examples/pytorch/mnist_tensorboard_artifact.py

```diff
@@ -218,3 +220,23 @@ def log_scalar(name, value, step):
         "\nLaunch TensorBoard with:\n\ntensorboard --logdir=%s"
         % os.path.join(mlflow.get_artifact_uri(), "events")
     )
+
+    # Log the model as an artifact of the MLflow run.
+    print("\nLogging the trained model as a run artifact...")
+    mlflow.pytorch.log_model(model, artifact_path="pytorch-model", pickle_module=pickle)
+    print(
+        "\nThe model is logged at:\n%s" % os.path.join(mlflow.get_artifact_uri(), "pytorch-model")
+    )
+
+    # Since the model was logged as an artifact, it can be loaded to make predictions
+    loaded_model = mlflow.pytorch.load_model(mlflow.get_artifact_uri("pytorch-model"))
+
+    # Extract a few examples from the test dataset to evaulate on
+    _, (eval_data, eval_labels) = next(enumerate(test_loader))
```

Comment:
Can we remove `enumerate` here?

Reply:
I used enumerate to get a single batch from the test loader. Shall I replace it with `iter`? Is there a reasoning that I am missing?

Reply:
`iter` looks better to me because we can remove `_`.

Reply:
Ahh got you. Thanks!